lib/maintenance_tasks.rb

```diff
@@ -9,6 +9,10 @@
 require 'pagy'
 require 'pagy/extras/bulma'
 
+# Force the TaskJob class to load so we can verify upstream compatibility with
+# the JobIteration gem
+require_relative '../app/jobs/maintenance_tasks/task_job'
```

Comment:
Maybe instead of requiring the file we could to the check here instead? But that separates it from the patch itself so it's probably worse.

Reply:
Yeah I think it might be less confusing to keep them altogether for now